src/ngx_http_modsecurity_module.c

```diff
@@ -766,5 +791,50 @@ ngx_http_modsecurity_cleanup_rules(void *data)
     ngx_http_modsecurity_pcre_malloc_done(old_pool);
 }
 
+static int
+ngx_http_modsecurity_set_up_log_reload(ngx_conf_t *cf, ngx_http_modsecurity_conf_t *mcf)
+{
+    ngx_http_modsecurity_main_conf_t  *mmcf;
+    ngx_list_t                        *list;
+    Rules                         **item;
+    
+    mmcf = ngx_http_conf_get_module_main_conf(cf, ngx_http_modsecurity_module);
+    list = mmcf->audit_log_reload->data;
+
+    // Each rules set may have an audit log. We need to remember each rules set
+    // so we can ask for its audit log to be reopened.
+    item = ngx_list_push(list);
+    if (item == NULL) {
+        dd("failed to append a rules set for log reload");
+        return -1;
+    }
+    *item = mcf->rules_set;
+
+    return 0;
+}
+
+static void
+ngx_http_modsecurity_log_reload(ngx_open_file_t *file, ngx_log_t *log)
```

Comment:
We should be using the word "reopen" rather than "reload" for these functions and this functionality.

Reply:
Renamed to use "reopen" rather than "reload" in 2b36285